app/scripts/views/tour/tour-controller.js

```diff
@@ -5,12 +5,19 @@
      * Controller for the gw app tour view
      */
     /* ngInject */
-    function TourController() {
+    function TourController($log, $scope, Config, Visualization) {
         var ctl = this;
         initialize();
 
         function initialize() {
-            cartodb.createVis('map', 'http://greenworks.cartodb.com/api/v2/viz/33627780-f988-11e4-8ff7-0e0c41326911/viz.json');
+            var vis = cartodb.createVis('map', 'http://greenworks.cartodb.com/api/v2/viz/33627780-f988-11e4-8ff7-0e0c41326911/viz.json');
+            Visualization.add(Config.ids.tour, vis);
+
+            // Delete vis on scope destroy to free resources
+            // and allow for it to be created again later
+            $scope.$on('$destroy', function () {
+                Visualization.remove(Config.ids.tour);
```

Comment:
Seems like something more should happen than removing it from the collection of visualizations.  Will this remove it from the map?  Does it destroy the map?